services/download/src/main/java/app/coronawarn/server/services/download/config/DownloadServiceConfigValidator.java

```diff
@@ -0,0 +1,49 @@
+
+
+package app.coronawarn.server.services.download.config;
+
+import java.util.Map;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+/**
+ * Validate the values of the DownloadServiceConfig.
+ */
+public class DownloadServiceConfigValidator implements Validator {
```

Comment:
I think you need to register this validator bean statically before the config properties get injected in the DownloadServiceConfig. See how the SubmissionServiceValidator is registered in the Application.class of the submission module

Reply:
Correct. Without the static bean definition the validation is not executed. Did the change as requested.